pylot/drivers/carla_lane_invasion_sensor_operator.py

```diff
@@ -41,17 +35,13 @@ def __init__(self, ground_vehicle_id_stream: erdos.ReadStream,
         self._lane_invasion_sensor = None
         self._map = None
 
-    @staticmethod
-    def connect(ground_vehicle_id_stream: erdos.ReadStream):
-        lane_invasion_stream = erdos.WriteStream()
-        return [lane_invasion_stream]
-
-    def run(self):
+    def run(self, read_stream, write_stream):
         # Read the vehicle ID from the vehicle ID stream.
-        vehicle_id_msg = self._vehicle_id_stream.read()
-        vehicle_id = vehicle_id_msg.data
-        self._logger.debug("@{}: Received Vehicle ID: {}".format(
-            vehicle_id_msg.timestamp, vehicle_id))
+        vehicle_id = read_stream.read()
+
+        self._logger.debug(
+            "The LaneInvasionSensorOperator received the vehicle id: {}".
+            format(vehicle_id))
```

Comment:
Could you revert the error message to what it was before? We just fixed a bug in ERDOS, which resulted in `read_stream.read()` returning data instead of messages. You may need to pull and reinstall the latest ERDOS version.
```suggestion
        self._logger.debug("@{}: Received Vehicle ID: {}".format(
            vehicle_id_msg.timestamp, vehicle_id))
```